.github/workflows/ci-postgres-mysql.yml

```diff
@@ -34,6 +34,33 @@ jobs:
           path: ./packages/**/dist
           key: ${{ github.sha }}:db-tests
 
+  sqlite-pooled:
```

Comment:
The file name no longer matches all the contents, but I think it's fine

Reply:
didn't want to rename the file, and create a new entry a new workflow in the actions.